t/macros/load_macros.t

```diff
@@ -34,7 +34,17 @@ my %baseMacros = (
 );
 
 # PG_CAPAmacros.pl is not really broken, but it depends on files that are in the OPL and not in the PG repository.
-my %brokenMacros = ('answerDiscussion.pl' => 1, 'PG_CAPAmacros.pl' => 1);
+# PGplot helper macros Axes.pl, GD.pl, Tikz.pl, and Data.pl are not really broken, but shouldn't be loaded directly.
+# PGplotmacros.pl overrides some default WWplot objects. Loading this causes other test to fail.
+my %brokenMacros = (
+	'answerDiscussion.pl' => 1,
+	'PG_CAPAmacros.pl'    => 1,
+	'Axes.pl'             => 1,
+	'GD.pl'               => 1,
+	'Tikz.pl'             => 1,
+	'Data.pl'             => 1,
+	'PGplotmacros.pl'     => 1,
```

Comment:
Would modules in `lib` be preferable over combining `Axes.pl`, etc in a single file? I just feel it would be better to have everything loaded from a macro as needed, but it probably doesn't matter.

I'll push `PGplotmacros.pl` to a second PR. The current way it is setup, it modifies the WWplot methods, causing other macros to fail their tests, and PGplot can be tested without this wrapper.